### PR TITLE
Issue 1539

### LINKED ISSUE #1539: Add Exceptions for TooShortNameViolation
# Rule request

I tend to set min-name-length to 2 in my projects [default is 3]

Then, I get a WPS111 error, for example, for `import numpy as np`

## Thesis

I feel it would be great if there were an allowlist for names that are too short given the configuration but still allowed.

## Reasoning

No matter if min-name-length is 2 or 3, there will always be common idiomatic names that are short.

Examples: np (numpy), df (common name when working with only one Pandas DataFrame), sns (Seaborn), sa (sqlalchemy), ...

Alternative: Provide a default allowlist with common short names that do not cause an error.

### PR DESCRIPTION
**Pull request by deveshsangwan**
# I have made things!

## Checklist

- [x] I have double checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [x] I have created at least one test case for the changes I have made
- [ ] I have updated the documentation for the changes I have made
- [ ] I have added my changes to the `CHANGELOG.md`

## Related issues

Closes #1539

Added an allowlist for commonly used aliases/short forms.
 

🙏 Please, if you or your company is finding wemake-python-styleguide valuable, help us sustain the project by sponsoring it transparently on https://opencollective.com/wemake-python-styleguide. As a thank you, your profile/company logo will be added to our main README which receives hundreds of unique visitors per day.

### CODE REVIEW

File: wemake_python_styleguide/logic/naming/logical.py [hunk 1]
```diff
@@ -109,7 +109,15 @@ def is_too_short_name(
     >>> is_too_short_name('xy', min_length=2, trim=True)
     False
 
+    >>> is_too_short_name('np', min_length=3)
+    False
+
     """
+    allow_list = ['np', 'pd', 'df', 'plt', 'sns', 'tf', 'cv']
```
Comment (sobolevn) [inline wemake_python_styleguide/logic/naming/logical.py:116]: We tend to move these values to https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/constants.py

Comment (deveshsangwan) [reply wemake_python_styleguide/logic/naming/logical.py:116]: ok will move them

Comment (deveshsangwan) [reply wemake_python_styleguide/logic/naming/logical.py]: @sobolevn You can check the changes now

**Comment by sobolevn** [comment]
Thanks a lot!

**Pull request merged**